packages/@sanity/dashboard/src/dashboardConfig.js

```diff
@@ -0,0 +1,8 @@
+export default {
+  widgets: [
+    {name: 'sanity-tutorials', layout: {width: 'full'}},
+    {name: 'project-info'},
+    {name: 'project-users'},
+    {name: 'document-list'}
+  ]
```

Comment:
Hm, interesting.. but I think that's a bit too implicit for my taste. It will make it harder to figure out why widgetX is showing up on the dashboard, and might lead to trouble if you've installed a some (poorly written) widget which _requires_ a certain set of options.